BuildScript.UnitTests/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="2.6.2" targetFramework="net45" />
+  <package id="NUnit" version="3.12.0" targetFramework="net47" />
```

Comment:
This means we can only use this once all build agents are updated to .net 4.7 -> dependency on the docker-task @iam-mholle

Reply:
We have 3.12  here and 3.11 when it comes to the NUnit runner. Is that an oversight?

Reply:
No, it's just the way NUnit released their stuff. The newest NUnit "types" are at 3.12 while the newest NUnit console runner is at 3.11.